tool.go

```diff
@@ -750,7 +750,17 @@ func runNode(script string, args []string, dir string, quiet bool) error {
 	}
 
 	if runtime.GOOS != "windows" {
-		allArgs = append(allArgs, "--stack_size=10000", script)
+		r := syscall.Rlimit{}
```

Comment:
Since you don't care about initial value (i.e., you want zero, it will be populated in next line), use `var r syscall.Rlimit` form. It's slightly more common in this type of situation.